tests/jobs/rft/test_trajectory.py

```diff
@@ -102,6 +106,29 @@ def test_load(initdir, fname):
     ):
         assert trajectorypoint.utm_x == expected_utmx
 
+    dframe = trajectory.to_dataframe()
+    assert isinstance(dframe, pd.DataFrame)
+    assert len(dframe) == 2
+    # Check that we have the input columns:
+    assert {"utm_x", "utm_y", "measured_depth", "true_vertical_depth", "zone"}.issubset(
+        set(dframe.columns)
+    )
+    # grid_ijk is a temp column, not to be present in output
+    assert "grid_ijk" not in dframe
+
+    # pressure should not be there when there is no data for it:
+    assert "pressure" not in dframe
```

Comment:
This could probably be it´s own test with one test for `pressure` in dframe, and one without. That would help me in knowing when to expect `pressure`, and when not to expect it.

Reply:
The test with `pressure` is in `test_gendata_rft.py`. In `test_trajectory.py` there is only clean trajectories. Clearified in docstrings and comments.